packages/@cruise-automation/worker-rpc/README.md

```diff
@@ -0,0 +1,74 @@
+# @cruise-automation/worker-rpc
+
+An RPC layer to make it easier to communicate between a WebWorker and the main thread.  Has support for sending and responding with [transferable](https://developer.mozilla.org/en-US/docs/Web/API/Transferable) objects to avoid structured cloning of large array buffers.  It also propagates errors thrown in receivers back to the calling thread.
```

Comment:
`An RPC...` => `A RPC...`. 
I saw that you referred `WebWorker(s)` a few times in the doc text, I'm wondering if we can just call it `web workers` so it's easily to associate with native web worker concept.